metamer/ftest/src/test/java/org/richfaces/tests/metamer/ftest/utils/ManagementClientProvider.java

```diff
@@ -0,0 +1,34 @@
+package org.richfaces.tests.metamer.ftest.utils;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.wildfly.extras.creaper.core.ManagementClient;
+import org.wildfly.extras.creaper.core.offline.OfflineManagementClient;
+import org.wildfly.extras.creaper.core.offline.OfflineOptions;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.OnlineOptions;
+
+/**
+ * Provider for Creaper's OnlineManagementClient
+ */
+public class ManagementClientProvider {
+
+    /**
+     * Creates OnlineManagementClient for standalone server
+     * @return Initialized OnlineManagementClient, don't forget to close it
+     */
+    public static OnlineManagementClient createOnlineManagementClientForStandaloneServer() {
+        return ManagementClient.onlineLazy(OnlineOptions.standalone()
+                .hostAndPort("localhost", 9990)
+                .build());
```

Comment:
What about older JBoss AS7 (EAP 6)? Is there used different branch for its testing?